src/Psalm/Config.php

```diff
@@ -559,10 +559,7 @@ class Config
      */
     public $debug_emitted_issues = false;
 
-    /**
-     * @var bool
-     */
-    private $report_info = true;
+    private bool $report_info = true;
```

Comment:
Not sure how this differs from, say, `$max_string_length`. Why did one get promoted to native types and another did not?

Reply:
`$max_string_length` is public and would therefore be a BC break.

Reply:
Ok, makes sense.